### ISSUE TITLE
Suggestion: ability to change hidden comment color.

### ISSUE THREAD

**Issue by Niksharpkings** [estruyf/vscode-hide-comments#24]
I know it's a visual hidden trick, but when I tend to forget where that comment was and start writing code where to comment for another line of code, so I tried messing with the workspace settings json file:
```
 "settings": {
          "foreground": "#00000000"
        }
```

It would be nice if there could be a simple command or preferably and mainly the user setting where I set my own color or have an opacity range scale. 

Basically, what I'm trying to say is I don't want to hide the comment entirely, but just barely visible so I don't start writing code on the wrong line.
If it wouldn't be so much trouble, 
Thanks. love the extension so far.

**Comment by estruyf** [comment]
That could be an excellent enhancement to the extension. For example, we can add a `hideComments.commentsColor` setting which can be overwritten.

**Comment by estruyf** [comment]
Just released version 1.8.0 with a new `hideComments.color` setting.
